0032-peering-clearing-settlement/0032-peering-clearing-settlement.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Peering, Clearing and Settling on the Interledger network
+draft: 1
+---
+
+The Interledger network is a graph of nodes (connectors) that have peered with one another by establishing a means of exchanging ILP packets and a means of paying one another for the successful forwarding and delivery of the packets.
+
+Further details on the ILP Protocol can be found in the [specification](../0027-interledger-protocol-4/0027-interledger-protocol-4.md).
+
+# Accounts and Balances
+
+The edge between any two nodes (peers) is a communication link (for exchanging ILP packets and other data), and an account held between the peers (the Interledger account). The Interledger account has a balance denominated in a mutually agreed upon asset (e.g. USD) at an agreed upon scale (e.g. 2). The balance on the account is **the net total of the amounts on any packets successfully routed between the peers**.
+
+The balance on the Interledger account can only change as a result of two events:
+ 1. The successful routing of an ILP packet between the two peers
+ 1. A payment made between the peers on the underlying payment network they have agreed to use to settle the Interledger account
+
+ This document provides further details on the when these events occur andhow they are handled by the peers. 
+
+> **Example : Alice and Bob**
+
+> - Alice and Bob are peers on the network.  
```

Comment:
Nitpick: Are they peers before they set up their peering relationship? Before that they are just nodes on the Interledger network